source/jmap/types.d

```diff
@@ -1124,21 +960,13 @@ class FilterCondition : Filter {
     }
 }
 
-Filter operatorAsFilter(FilterOperator filterOperator) {
```

Comment:
this is still registered at https://github.com/symmetryinvestments/imap-d/pull/109/files#diff-534e4ee761393f362ae7ae3ca4c389bfd507a159a52529ae4bacc2032748f74dR15

Reply:
Thanks, removed from register log.